core/new-gui/src/app/workspace/component/operator-panel/operator-panel.component.ts

```diff
@@ -76,7 +75,9 @@ export class OperatorPanelComponent implements OnInit {
           return [];
         }
         // TODO: remove this cast after we upgrade to Typescript 3
-        const results = this.fuse.search(v) as OperatorSchema[];
```

Comment:
the return value of the search method has been changed, the attribute name containing the results is "item".
check https://fusejs.io/examples.html#search-string-array for more details about the format.